src/js/site/main.js

```diff
@@ -401,4 +402,41 @@ $(document).ready(function () {
 });
 
 // Currently this function is only used for the insights table
-$(document).ready(highlightURLHash());
+$(highlightURLHash);
+
+// update date shown to be X ago
+$(function () {
+  // tooltip code for posted on time
+  const tooltiptime = document.getElementById('tooltip-time');
+  let timeposted = document.getElementById('time-posted-on');
+  let popperInstance = null;
+  let showEvents = ['mouseenter', 'focus'];
+  let hideEvents = ['mouseleave', 'blur'];
```

Comment:
```suggestion
  const hideEvents = ['mouseleave', 'blur'];
```

Reply:
can you also move the `showEvents` and `hideEvents` on line 274 & 275 respectively to the upper scope and use them instead of re-declaring?